frontend/src/components/Pages/RobotPage/RobotArmMovement.tsx

```diff
@@ -0,0 +1,81 @@
+import { Autocomplete, AutocompleteChanges, Button, Card, Dialog, Typography, Icon } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { useState, useEffect } from 'react'
+import { BackendAPICaller } from 'api/ApiCaller'
+import { Robot } from 'models/Robot'
+
+const StyledDialog = styled(Card)`
+    display: flex;
+    padding: 1rem;
+    width: 600px;
+    right: 175px;
+`
+const StyledAutoComplete = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+`
+
+const StyledButtons = styled.div`
+    display: flex;
+    gap: 8px;
+    justify-content: flex-end;
+`
+
+interface RobotProps {
+    robot: Robot
+}
+
+export function BatteryPose({ robot }: RobotProps) {
+    const [feedback, setFeedback] = useState('')
+
+    const onClickBatteryPose = () => {
```

Comment:
Now there are two functions, one for battery pose and one for transport pose.

This should be one function which takes the pose as an input parameter. Then the same function may be used for two different buttons but the difference will be the input parameter to the function which is either "battery_change" or "transport"